### DISCUSSION TITLE
编译原理：语言与文法 · Cup Air

### DISCUSSION THREAD

**Discussion by ACupofAir** [ACupofAir/ACupofAir.github.io#18] [Announcements]
https://acupofair.github.io/posts/compile/lang_grammar/ 

 
🐉2021 年与龙书斗争的第一天与最后一天

**Comment by LeiBuDao** [comment]
狗王骏，居然背着我偷学！！！可恶！！！

**Comment by thisisbadBao** [comment]
> @LeiBuDao
> 狗王骏，居然背着我偷学！！！可恶！！！

我刚准备开始学呢😄

**Comment by LeiBuDao** [comment]
> @thisisbadBao
> > @LeiBuDao
> > 狗王骏，居然背着我偷学！！！可恶！！！
> 
> 我刚准备开始学呢😄
快学！
